components/support/crypto/nss-sys/src/types/nss3.rs

```diff
@@ -0,0 +1,460 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use super::nspr::*;
+use std::os::raw::{c_char, c_int, c_uint};
+
+#[repr(C)]
+pub enum SECStatus {
+    SECWouldBlock = -2,
+    SECFailure = -1,
+    SECSuccess = 0,
+}
+pub use SECStatus::*;
+
+#[repr(C)]
+pub enum SECOidTag {
+    SEC_OID_UNKNOWN = 0,
```

Comment:
Do we need to list all of these in order for the `repr(C)` representation to have the right shape, or could we get away with only listing the specific values that we need?

Reply:
We can list only what we need.